flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/JoinQueryOperation.java

```diff
@@ -106,6 +107,32 @@ public String asSummaryString() {
                 "Join", args, getChildren(), Operation::asSummaryString);
     }
 
+    @Override
+    public String asSerializableString() {
+        // TODO: correlated
```

Comment:
resolve TODO?

Reply:
actually I have, just forgot to remove the `TODO`